prboom2/src/p_setup.c

```diff
@@ -2641,13 +2641,13 @@ void P_SetupLevel(int episode, int map, int playermask, skill_t skill)
   // find map name
   if (gamemode == commercial)
   {
-    sprintf(lumpname, "map%02d", map);           // killough 1/24/98: simplify
-    sprintf(gl_lumpname, "gl_map%02d", map);    // figgi
+    snprintf(lumpname, 9, "map%02d", map);           // killough 1/24/98: simplify
```

Comment:
Pedantic, but could you use `sizeof()` please?

Reply:
Sure, I'll do it that way. A new version has been added with that change applied.